cmd/lncli/cmd_walletunlocker.go

```diff
@@ -754,11 +758,18 @@ func createWatchOnly(ctx *cli.Context) error {
 		}
 	}
 
-	_, err = client.InitWallet(ctxc, &lnrpc.InitWalletRequest{
+	statelessInit := ctx.Bool(statelessInitFlag.Name)
```

Comment:
should we do a similar check?
```go
statelessInit := ctx.Bool(statelessInitFlag.Name)
if !statelessInit && ctx.IsSet(saveToFlag.Name) {
	return fmt.Errorf("cannot set save_to parameter without " +
		"stateless_init")
}
```

Reply:
Good point, thanks! Also fixed the missing error return (and a deprecated call).